rust-mode.el

```diff
@@ -186,6 +186,7 @@ function or trait.  When nil, where will be aligned with fn or trait."
   "Format future rust buffers before saving using rustfmt."
   :type 'boolean
   :safe #'booleanp)
+(put 'rust-format-on-save 'safe-local-variable 'booleanp)
```

Comment:
Isn't this line redundant given that the line above expresses exactly the same thing (in a nicer way)?

`defcustom`'s documentation says:

```
...
:safe	Set SYMBOL's `safe-local-variable' property to VALUE.
        See Info node `(elisp) File Local Variables'.
...
```